### ISSUE TITLE
@require'd component not passing into exported function?

### ISSUE THREAD

**Issue by l0oky** [jaredhanson/electrolyte#17]
OK, this is maybe only  a newbies mistake. The thing is I can't see my component 'models/procedures' in my another component 'controllers/uploadHandler' [here's an example.](https://gist.github.com/l0oky/c07bbf08cde4bd866082)

What did I do wrong?

Electrolyte is awesome by the looks of it. Very simple concept and powerful, but lacks proper documentation ( I guess ). Anyhow I started learning it by reading the [readme](https://github.com/jaredhanson/electrolyte/blob/master/README.md) and following [jsummers electrolyte example.](https://github.com/jsumners/njs-di-example) ( I thank him very much, learn a lot from that. )

Basically I wanted to seperate database procedures and controllers. 'config/database' returns a connection of a postgres database and 'models/procedures' uses that connection. Other component like that 'controllers/uploadHandler' use functions that are in 'models/procedures', and that's my idea on how to communicate with a database using clean code thanks to electrolyte. :+1: 

Thank you :)

**Comment by l0oky** [comment]
I didn't return an instance in 'models/procedures'. It's one of those things that you miss. Sorry for polluting your repo.

**Comment by jsumners** [comment]
This should be cleared up -- https://github.com/jsumners/njs-di-example/issues/1#issuecomment-87672227
